wolfssl/wolfcrypt/asn.h

```diff
@@ -707,6 +707,8 @@ enum DN_Tags {
     ASN_DNQUALIFIER   = 0x2e,   /* dnQualifier */
 #endif /* WOLFSSL_CERT_NAME_ALL */
 
+    ASN_CONTENT_TYPE  = 0x03,   /* pkcs9_contentType */
```

Comment:
This doesn't conflict with `ASN_COMMON_NAME   = 0x03,   /* CN */` ?

Reply:
I think both are `0x03`. Oid defined here https://datatracker.ietf.org/doc/html/rfc2630#section-11.1

Reply:
This was spot on. Their identifier id's were the same but different base OID's. Fixed in the second commit.